src/fdcache_entity.cpp

```diff
@@ -1476,23 +1476,28 @@ bool FdEntity::MergeOrgMeta(headers_t& updatemeta)
     if(0 != updatetime){
         SetCtime(updatetime, true);
     }
-    bool is_pending = !upload_id.empty();
+    is_meta_pending = !upload_id.empty();
```

Comment:
If is_meta_pending is true, it may be rewritten to false.
I think it should be: 
`is_meta_pending |= ! upload_id.empty ();`

Reply:
accept